quipucords/api/details_report/serializer.py

```diff
@@ -1,6 +1,11 @@
 """Serializer for system facts models."""
 
-from rest_framework.serializers import CharField, IntegerField, JSONField, UUIDField
+from rest_framework.serializers import (
+    CharField,
+    IntegerField,
+    JSONField,
+    UUIDField,
+)
```

Comment:
What touched this line? ruff seemed happy with the previous version.

Reply:
🤔 During development I added some code here and then removed. I guess black (I was originally using black) formatted the extra dependencies like this and when removed left it in this format. Will adjust it